src/object-spec-code-utils.ts

```diff
@@ -46,7 +46,17 @@ function typeForUnderlyingType(underlyingType:string):ObjC.Type {
   };
 }
 
-export function computeTypeOfAttribute(attribute:ValueObject.Attribute):ObjC.Type {
+function propertyModifierForCopyable(supportsValueSemantics: boolean):ObjC.PropertyModifier {
+  // If generated object supports value semantics we should return Copy attribute.
+  // Otherwise we want it to be assigned
```

Comment:
The code below pretty much says exactly this. I would remove the comment. Comments are not executable and will potentially drift from the implementation given they don't have to be updated